In [1]:
import pandas as pd

In [3]:
survey_results = pd.read_csv("/Users/alexmahadevan/Desktop/Projects/Survey/SURVEY_WITH_DUMMIES.csv")

In [5]:
survey_results.head(1)

Response_ID  Time_Started Date_Submitted    Status  Contact ID  \
0            1  7/24/17 4:36   7/24/17 4:48  Complete         NaN   

   Legacy Comments  Comments Language  \
0              NaN       NaN  English   

                                             Referer  \
0  http://www.surveygizmo.com/s3/3720823/Work-Fro...   

                                    Extended_Referer  \
0  http://www.surveygizmo.com/s3/3720823/Work-Fro...   

                                       ...                                       \
0                                      ...                                        

  benefits_parental_leave benefits_401k benefits_discounts  types_of_wfh_jobs  \
0                     NaN           NaN  Product discounts      No preference   

  accommodate_vision  accommodate_hearing  accommodate_cognitive  \
0                NaN                  NaN                    NaN   

  accommodate_physical visit_jobs_page  \
0                  NaN             0.0   

  Please offer any suggestions you have for jobs coverage at The Penny Hoarder.  
0  You actually do a pretty good job. It woukd be...                             

[1 rows x 76 columns]

<p>Let's create a new dataframe that drops all of the partial responses.</p>
<p>That way we don't get tripped upo with too many blank responses.</p>

In [6]:
survey_results_complete = survey_results[survey_results.Status != 'Partial']

In [35]:
survey_results_complete.Status.value_counts()

Complete    2017
Name: Status, dtype: int64

<p>Now, let's break the dataframe into one for just men and one for just women.</p>

In [36]:
men = survey_results_complete[survey_results_complete.Gender == '0']

In [38]:
women = survey_results_complete[survey_results_complete.Gender == '1']

<p>Let's look at the disparity in salary demands between men and women.</p>

In [41]:
men.pay_preferred.value_counts().reset_index()

index  pay_preferred
0  $15/hour or greater            108
1         $10-$12/hour             67
2         $12-$14/hour             53
3          $8-$10/hour             31
4        Up to $8/hour              2

In [42]:
women.pay_preferred.value_counts().reset_index()

index  pay_preferred
0  $15/hour or greater            591
1         $10-$12/hour            512
2         $12-$14/hour            361
3          $8-$10/hour            240
4        Up to $8/hour             38

In [56]:
male_pay = men.pay_preferred.value_counts().reset_index()

In [57]:
female_pay = women.pay_preferred.value_counts().reset_index()

In [62]:
male_pay['percent'] = male_pay['pay_preferred'] / 261

In [63]:
male_pay.head()

index  pay_preferred   percent
0  $15/hour or greater            108  0.413793
1         $10-$12/hour             67  0.256705
2         $12-$14/hour             53  0.203065
3          $8-$10/hour             31  0.118774
4        Up to $8/hour              2  0.007663

In [64]:
female_pay['percent'] = female_pay['pay_preferred'] / 1742

In [65]:
female_pay.head()

index  pay_preferred   percent
0  $15/hour or greater            591  0.339265
1         $10-$12/hour            512  0.293915
2         $12-$14/hour            361  0.207233
3          $8-$10/hour            240  0.137773
4        Up to $8/hour             38  0.021814

<p>Here you see that 41% of men demanded the highest wage, while only 34% of women did that.</p> Men also had a smaller preference for low wages. 12% of men said they would work for $10 or less, while 16% of women said they would.</p>